java/AxivityReader.java

```diff
@@ -7,7 +7,11 @@
 import java.nio.channels.FileChannel;
 import java.nio.channels.ReadableByteChannel;
 import java.time.Duration;
+import java.time.LocalDate;
 import java.time.LocalDateTime;
+import java.time.ZoneId;
```

Comment:
@chanshing - do you think we can move these new import statements to the abstract DeviceReader.java class?

i.e. we would like these to be available to all of AxivityReader.java, ActigraphReader.java, etc.